### PR TITLE
add new xml wrapper with the modified help section

### PR DESCRIPTION
**Pull request by charles-bernard**
just changed the help section

### CODE REVIEW

File: tools/trim_galore/trim_galore.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
 <tool id="trim_galore" name="Trim Galore!" version="0.4.2">
```
Comment (bgruening) [inline tools/trim_galore/trim_galore.xml]: I guess we should bump the version number to 0.4.3 or 0.4.2.1.

**Comment by bgruening** [comment]
Charles thanks a lot!
Are you familiar with conda packages? If so are you interested in getting biskark under continues testing?

**Comment by charles-bernard** [comment]
You're welcome Björn ! To be honest, I'm not familiar at all with conda packages. What are they meant for (in the context of testing) ?

**Comment by bgruening** [comment]
Conda is the new endorsed package manager of Galaxy. So if a tool is in bioconda Galaxy will use this package to run your tool - and this means you can test all tools on github. No more tool_dependencies.xml .... Conda is the future here :-)

**Comment by bgruening** [comment]
This is great! Help improvements are so much needed for my tools! Thanks a bunch!

**Comment by nsoranzo** [comment]
@bgruening Can you push this to the MTS? It seems the automatic update failed.

**Comment by bgruening** [comment]
Done. The automatic update does not work with bgruening/rnatema in one repo for some reasons.

**Comment by nsoranzo** [comment]
Thanks!

**Pull request merged**